New evaluation.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>New Evaluation</title>
+    
+    
+<link rel="stylesheet" href="css/New%20Evaluation.css">
+</head>
+<body>
+    
+    <form id="candidate">
+        <input type="text" placeholder="Candidate" name="Candidate">
+        <input type="text" placeholder="Interviewer" name="Interviewer">
+        <input type="date">
+    </form>
+    
+    <h1>Techinical Level</h1>
+<form>
+    <table align="centre">
+        <tr>
+            <th>Trainee</th>
+            <th>Junior</th>
+            <th>Mid</th>
+            <th>Senior</th>
+        </tr>
+        <tr>
+            <td>
+                <input type="radio" name="level" value="1"></td>
+            <td>
+                <input type="radio" name="level" value="j1">
+                <input type="radio" name="level" value="j2">
+                <input type="radio" name="level" value="j3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="m1">
+                <input type="radio" name="level" value="m2">
+                <input type="radio" name="level" value="m3">
+            </td>
+            <td>
+                <input type="radio" name="level" value="s1">
+                <input type="radio" name="level" value="s2">
+                <input type="radio" name="level" value="s3">
+            </td>
+        </tr>
+        
+    
+    </table>
+</form>
+    
+    <h2>Should the candidate be hired?</h2>
+    <form>
+        <textarea rows="5" cols="30" placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance required for the candidate"></textarea>
+    </form>
+    
+    <h2>General Impression</h2>
+    <form>
+        <textarea rows="5" cols="30" placeholder="*required"></textarea>
+    </form>
+    
+    <h2>Workflow, Leadership &#38 Soft Skills</h2>
+    <form>
+       <textarea rows="5" cols="30" placeholder="Describe the environment in which the candidate works.&#13;&#10;Describe any guidance or management experience"></textarea>
+    </form>
+    
+    <h2>OOP, Design Patterns</h2>
+    <form method="post" id="OOP">
+        <ul class="clearfix">
+            <li>Classes</li>
+            <li>
+                <select>
+                <option selected disabled hidden>Evaluation</option>
+                <option name="Classes" value="0">0</option>
+                <option name="Classes" value="1">1</option>
+                <option name="Classes" value="2">2</option>
+                <option name="Classes" value="3">3</option>
+                </select>
+            </li>
+            <li>Exception handling</li>
+            <li>
+                <select>
+                <option selected disabled hidden>Evaluation</option>
```

Comment:
as children for the select tag, those options will require an increase in the indent level; readability is a very important aspect of writing code in a team... you have to make sure that the code you write is easily understandable by your team members.